### ISSUE TITLE
[REQ] more downloaders, more fun

### ISSUE THREAD

**Issue by forart** [shirooo39/MiXLab#7]
1st of all thanks for your GREAT collection of extremely useful tools that helpd us to solve *a lot* of problems.

Request: is possible to add/implement those scripts too ?

https://github.com/dhhruv/YouTua
https://github.com/iheanyi/bandcamp-dl
https://github.com/setnicka/ulozto-downloader
https://github.com/h4ckninja/twitter-video-downloader

Thanks in advance !

**Comment by shirooo39** [comment]
For YouTua and twitter-video-downloader, there's already Youtube-DL implemented (credit to... whoever wrote the code, really sorry if I didn't remember / know who wrote the code). Though, I'm not sure if Youtube-DL is able to download from twitter or not, but you can check their supported sites.

For the ulozto-downloader, what is that? a multi-downloader? there's already aria2 that is capable of downloading both hosted files and p2p, basically already multi-downloader.

For the bandcamp-dl, since it's a python script, I think it should be easy to add. still not sure how I should do it, but I'll look into it when I have the time...

**Comment by forart** [comment]
You're right, yt-dl supports almost everything btw the YouTua interface is nice.

Uloz.to - that means means "save it" - is a Czech file sharing service founded in 2007: https://uloz.to/
A collaboration with script author @setnicka would be great to implement it in colab.

Of course, the same for @iheanyi for bandcamp-DL...

**Comment by shirooo39** [comment]
For now, I've added an extremely simple implementation of bandcamp-dl that you can test in the "EXPERIMENTAL" area or you can click on this link (https://colab.research.google.com/github/shirooo39/MiXLab/blob/master/MiXLab.ipynb#scrollTo=N09EnjlB6wuV) to jump right into it.

The URL and Download_location fields are working but for the other options, only "verbose logging" that I enabled for now.

Since I have zero knowledge of this bandcamp site, it might take me some times to enable the other options as I'd also need to do the testing as well.

**Comment by shirooo39** [comment]
I just updated the bandcamp-dl cell and moved it to the downloader section, out of the experimental area.
still not yet finished, but I think it should be useable at this state.

please do more testing.
also, the last two options are still not used yet (Specify_allowed_characters_in_slugify and Specify_the_character_to_use_in_place_of_spaces) so the default value from the script will be used instead.

**Comment by forart** [comment]
OK, just tested bandcamp-dl and seems to work correctly !

**Comment by forart** [comment]
...any chance for Uloz.to downloader ?

**Comment by forart** [comment]
1fichier:
- https://github.com/justfoolingaround/1fichier-utils
- https://github.com/micky008/unfichiergrabber

**Comment by forart** [comment]
> ...any chance for Uloz.to downloader ?
>
Ulozto downloader author (@setnicka) seems interested in porting his script to MixLab:
https://github.com/setnicka/ulozto-downloader/issues/54

**Comment by forart** [comment]
Ok, more 1fichier scripts:
- [1f-dl](https://github.com/devporter007/1f-dl) by @devporter007
- [pyOneFichierClient](https://github.com/spookyahell/pyOneFichierClient) by @spookyahell
- [upload-1fichier.com](https://github.com/MrAuronTV/upload-1fichier.com) by @MrAuronTV
- [weebseed_1fichier](https://github.com/zodman/weebseed_1fichier) by @zodman

- [down.med](https://github.com/fnjeneza/down.med) by @fnjeneza: uptobox and 1fichier downloader

Hope that inspires !

**Comment by forart** [comment]
Just discovered in @mohitjoshi155 - the [Codemaster](https://colab.research.google.com/github/mohitjoshi155/Codemaster/blob/master/All_in_One_1_9_1.ipynb) author - [repositories](https://github.com/mohitjoshi155?tab=repositories)

https://github.com/Gryffindor8/Netflix-videos-downloader
